include/mcptam/EntropyComputation.h

```diff
@@ -0,0 +1,44 @@
+/****************************************************************************************
+ *
+ * \file EntropyComputation.h
+ * \brief Functions for entropy computations
+ *
+ * Copyright 2015   Arun Das, University of Waterloo (adas@uwaterloo.ca)
+ * 
+ *
+ ****************************************************************************************/
+
+#include <ros/ros.h>
+#include <mcptam/Types.h>
+#include <mcptam/MapPoint.h>
+#include <mcptam/KeyFrame.h>  // needed for LEVELS define
+#include <vector>
+#include <boost/intrusive_ptr.hpp>
+#include <mcptam/TrackerData.h>
+#include <mcptam/MapPoint.h>
+#include <mcptam/Map.h>
+#include <mcptam/Utility.h>
+#include <mcptam/TrackerState.h>
+#include <mcptam/Tracker.h> 
+
+#ifndef __ENTROPY_COMPUTATION_H
+#define __ENTROPY_COMPUTATION_H
+
+
```

Comment:
I hate to say it, but the current code is pretty good about using brief and param on most methods in the headers. This hooks into the doxygen documentation.